meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "module_id": "viam:viam-telegraf-sensor",
   "visibility": "public",
-  "url": "https://github.com/viamrobotics/viam-telegraf-sensor",
+  "url": "https://github.com/viam-modules/viam-telegraf-sensor",
```

Comment:
good catch, just changed them. still now super familiar with how that works, I assume it will update everything as needed tho?

Reply:
hmm when I click the link the page can't be found

Reply:
but if I change it back to viamrobotics, it's no longer a link to a go pkg online

Reply:
neither `github.com/viamrobotics/viam-telegraf-sensor/telegrafsensor` nor `github.com/viam-modules/viam-telegraf-sensor/telegrafsensor` work for me.

Idk why it has the subpath /telegrafsensor

Can we just change it to a working link `github.com/viam-modules/viam-telegraf-sensor`

Reply:
I assume the viamrobotics version used to work but no longer does because I've moved the repo out of the viamrobotics org and into the viam-modules org. when I try the link you suggested there's error's and lint re-adds telegrafsensor to the end

Reply:
https://stackoverflow.com/questions/43716691/how-to-publish-a-go-package

reading this

Reply:
nice catch @hexbabe